lib/travis/build/addons/coverity_scan.rb

```diff
@@ -51,7 +51,7 @@ def authorize_quota
   else
     WHEN=`echo $AUTH_RES | ruby -e "require 'rubygems'; require 'json'; puts JSON[STDIN.read]['next_upload_permitted_at']"`
     echo -e "\033[33;1mCoverity Scan analysis NOT authorized until $WHEN.\033[0m"
-    exit 1
+    exit 0
```

Comment:
@BanzaiMan per the nested if on L44 there's exit 1, fallthrough, and exit 0... are we looking at the same thing?